### PR TITLE
Initial version of our Manifesto

### PR DESCRIPTION
**Pull request by Zakay**
We have had several workshop and group meetings, from that we have had comments that could become our manifesto. Here is a gathered version of our notes. Please review, add comments, simplify. I feel it might be too much, maybe some sentences can be simplified and shortened, maybe some things are repeated. Lets discuss.

### CODE REVIEW

File: MANIFESTO.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+As Hitta.se's development team, we believe in...
+
+1. **Testing matters**
+   - We exponentially increase the amount of rework when we try to find and fix bugs later in any lifecycle.
+   - That's why we test early and test often.
+   - We always dedicate time to expand our automated test suites by building both unit and end-to-end tests.
+2. **Quality above all**
+   - Quality takes priority over features and time.
+   - Quality has many facets, such as security, performance and even a great user experience.
+   - Code we write in the morning may be in production by the afternoon.
```
Comment (thomas-brx) [inline MANIFESTO.md]: Not sure this goes hand-in-hand with Quality, perhaps more related to "Automation is essential"?

File: MANIFESTO.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+   - We do continuous code reviews.
+   - We commit and push frequently.
+   - We give honest feedback.
+   - Processes are a means to help individuals and interactions.
+   - Learn from others and help others learn from you.
+4. **Coding is an art**
+   - Designing solutions for the right scale, don't over-architect.
+   - Aim for simplicity, honor KISS principle.
+   - Design patterns are inspirations, not solutions.
+   - You code for people not machines.
```
Comment (thomas-brx) [inline MANIFESTO.md]:
```suggestion
   - You code for people, not machines.
```

File: MANIFESTO.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+   - We make sure knowledge is shared.
+   - We do continuous code reviews.
+   - We commit and push frequently.
+   - We give honest feedback.
+   - Processes are a means to help individuals and interactions.
+   - Learn from others and help others learn from you.
+4. **Coding is an art**
+   - Designing solutions for the right scale, don't over-architect.
+   - Aim for simplicity, honor KISS principle.
+   - Design patterns are inspirations, not solutions.
```
Comment (thomas-brx) [inline MANIFESTO.md:22]: - Aggregate data at or close to the source - transfer as little as possible

(I.e. - aggregate *in* MongoDB, not in a service using MongoDB)

File: MANIFESTO.md (hunk 1 above)
Comment (Zakay) [reply MANIFESTO.md]: Yes correct!

File: MANIFESTO.md (hunk 3 above)
Comment (Zakay) [reply MANIFESTO.md:22]: Added!

**Review by thomas-brx — APPROVED** [review]
👍

**Review by sebastianwiik — APPROVED** [review]
:+1:

**Comment by prengifo** [comment]
Looks good! 😃

File: MANIFESTO.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
+# Manifesto
+As Hitta.se's development team, we believe in:
+
+1. **Testing matters**
+   - Testing early and often exponentially decreases the amount of rework as compared to trying to find and fix bugs later in any lifecycle.
+   - That's why we test early and test often.
```
Comment (bevanjf) [inline MANIFESTO.md]: Do we need this?

File: MANIFESTO.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+   - We always dedicate time to expand our automated test suites by building both unit and end-to-end tests.
+2. **Quality above all**
+   - Quality takes priority over features and time.
+   - Quality has many facets, such as security, performance and even a great user experience.
+3. **No islands**
+   - We collaborate on design and specs.
+   - We make sure knowledge is shared.
+   - We do continuous code reviews.
```
Comment (bevanjf) [inline MANIFESTO.md]: Change to "We continuously review our code"

File: MANIFESTO.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+4. **Coding is an art**
+   - Designing solutions for the right scale, don't over-architect.
+   - Aim for simplicity, honor KISS principle.
+   - Design patterns are inspirations, not solutions.
+   - You code for people, not machines.
+   - Using a data driven approach to solve problems, e.g. use of profiling tools and other collected data points.
+   - Leaving code better than you found it.
+   - Aggregate data at or close to the source - transfer as little as possible.
+   - Keeping number of dependencies at a minimum.
+   - Be pragmatic, not perfectionist.
```
Comment (bevanjf) [inline MANIFESTO.md]: "not a perfectionist" #grammar

File: MANIFESTO.md [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+3. **No islands**
+   - We collaborate on design and specs.
+   - We make sure knowledge is shared.
+   - We do continuous code reviews.
+   - We commit and push frequently.
+   - We give honest feedback.
+   - Processes are a means to help individuals and interactions.
+   - Learn from others and help others learn from you.
+4. **Coding is an art**
+   - Designing solutions for the right scale, don't over-architect.
```
Comment (bevanjf) [inline MANIFESTO.md]: The comma should be a full stop and Don't over-architect a new sentence #grammar

**Review by bevanjf — COMMENTED** [review]
Looks good, I left a few comments

**Pull request merged**